window.go

```diff
@@ -320,6 +325,10 @@ func (w *Window) SetVisible(visible bool) {
 			WindowManager().activateWindow(w)
 		}
 	}
+
+	if visible && w.onVisible != nil {
```

Comment:
1. PR description reads "Whenever a window has changed its visibility property, we call a user defined callback" but this code call a callback only when the Window gets visible. I proposed to use, e.g. field X (0: gets invisible, 1: gets visible). What do you think about it?
2. If we make the Window visible twice we get the event twice, too. Firing event should be inside `if w.hidden == visible {` block. Or add in the beginning  of the function:
```
if w.hidden != visible {
   return
}
```
then remove the condition below, unindent the block and keep your code as is